frontend/src/store/actions.js

```diff
@@ -40,15 +40,19 @@ export const actions = {
 
   async [LOGIN]({ commit, dispatch }, credentials) {
     commit(mutations.SET_LOGIN_PROGRESS, true);
+    commit(mutations.SET_LOGIN_ERROR, null);
     try {
       await dispatch(`http/${HTTP_LOGIN}`, credentials);
-      commit(mutations.SET_LOGIN_ERROR, null);
       dispatch(FETCH_USER, {
         callback: () => window.location.replace('/drive'),
       });
     } catch (err) {
       console.error(err);
-      commit(mutations.SET_LOGIN_ERROR, i18n.tc('login.login_error'));
+      const { response: {
+        status,
+      } } = err;
+
+      commit(mutations.SET_LOGIN_ERROR, status === 403 ? i18n.tc('login.login_group_error') : i18n.tc('login.login_error'));
```

Comment:
`403` also means wrong login/password

I found that's also problem on backend - in response details it has the warning about groups (Even if user does not exist!)

Reply:
Please check if you have the same error, we might need to create a ticket about it